.gitignore

```diff
@@ -53,3 +53,5 @@ Mkfile.old
 dkms.conf
 
 bin/
+
+vendor/
```

Comment:
You sure? We might need this if more Go code gets into the project

Reply:
Why would we need it, besides saving time on CI runs (and new downloads) for the initial `make vendor` ?

Reply:
If we ever grow Go bindings (there was some talk of moving some of the OCI code in here, I recall?) we'll probably be vendoring for that

Reply:
yeah and this also starts that work, but the vendoring can be deterministically created from just `go.mod` and `go.sum`. There doesn't seem like a need to pull the whole vendor directory into it if it can just be generated

Reply:
IMO

Reply:
Up to you - certainly keeps the repo cleaner